src/components/Markdown/Tiles.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'src/components/Link';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+
+export type TileProps = {
+  title: string;
+  description: string;
+  image: IconName;
+  link: string;
+};
+
+export const Tile = ({ title, description, image, link }: TileProps) => {
+  return (
+    <Link to={link} rel="noreferrer" className="block no-underline">
+      <section className="flex items-start gap-4 p-4 bg-neutral-100 rounded-lg border border-neutral-300 transition-colors hover:bg-neutral-200 hover:border-neutral-400 min-h-[106px]">
```

Comment:
We should include dark mode support in this! We don't use it yet, but if everything we build has support then it makes flicking the switch easier. For every colour (`XXX-neutral-YYY`), we should also add a corresponding dark class that flips around the other end of the colour palette. 

As an example, `bg-neutral-100` gains a corresponding `dark:bg-neutral-1200`. We have examples of this around the codebase. For more info on this, take a look at the [design reference](https://www.figma.com/design/jTQgyIovrhGBOf4Zrvjvbk/01---Styles?node-id=8066-308&t=4MSxbHduq5XawFFA-1)